ui/app/AppLayouts/Profile/views/SettingsContentBase.qml

```diff
@@ -161,7 +156,7 @@ FocusScope {
         // Left anchors and margin added bc of the implementation of the `SettingsContentBase` parent margin and to avoid
         // this toast to be wrongly centered
         anchors.left: root.left
-        anchors.leftMargin: -root.leftParentLayoutMargin / 2 + (root.width / 2 - width / 2)
```

Comment:
Can you add the comment in here to easily see why we are doing this "workaround" instead of just horizontal center?

Reply:
OK, added